code/components/dw-clipboard-input/dw-clipboard-input.js

```diff
@@ -30,20 +30,35 @@ class DwClipboardInput extends HTMLElement {
     this.onClipboardClick = async (event) => {
       const { inputElement, showToast } = this;
       const { value } = inputElement;
+
       event.stopPropagation();
       inputElement.select();
       inputElement.setSelectionRange(0, value.length);
-      document.execCommand("copy");
+      try {
+        document.execCommand("copy");
+      }
+      catch (err) {
+        // we ignore the error due to the fact that some browsers don't support one of the methods in the try block
+      }
+      try {
```

Comment:
aici nu mergea pus intrun singur try-catch?

Reply:
mi-a spus Cosmin sa las ambele variante total separat: intr-o parte cea deprecated (cu execCommand) si in cealalata parte cea actuala (cu clipboard.writeText)